src/parser/wsdl.js

```diff
@@ -21,6 +21,8 @@ var Schema = require('./xsd/schema');
 var Types = require('./wsdl/types');
 var Element = require('./element');
 
+var syncLoad;
```

Comment:
Why do we need this variable? Please note it's declared outside the `WSDL` class.